lib/octopolo/commands/pull_request.rb

```diff
@@ -3,6 +3,9 @@
   c.desc "Branch to create the pull request against"
   c.flag [:d, :dest, :destination], :arg_name => "destination_branch", :default_value => Octopolo.config.deploy_branch
 
+  c.desc "Pass -x to skip the prompt and infer from branch. Expects JIRA-123_describe_pr"
```

Comment:
OP is open source and this seems very SE specific.

Reply:
I would agree that it's JIRA specific but not SE specific.

Reply:
It's also hidden under an option, so it's opt in and not a default.

Reply:
Does this only skip the JIRA prompt, or all of the prompts? Oh I see. The branch name should be `"JIRA-123_describe_pr"`. Can you change the description to specify that what it expects is the branch?
```suggestion
  c.desc "Pass -x to skip the prompt and infer from branch. Expects the branch to be in this format: JIRA-123_describe_pr"
```